### PR TITLE
Feature/sub217

### PR DESCRIPTION
**Pull request by Haris188**

### CODE REVIEW

**Comment by Haris188** [comment]
@ynynl @l2silver, Code might look repetitive but in long run its not.

File: web-client/src/pages/organization/member/[route].tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import Header from 'shared/components/layout/Header'
+import DriversSearch from 'shared/components/drivers-search'
+import {
+    Container
+} from '@material-ui/core'
+
+const Dashboard: NextPage = ()=>{
+    const welcomeMsg = "Welcome back, Organization Name"
+    const components = [
+        {slug: 'drivers-search', label:'Driver Pre-Screen', component: <DriversSearch />}
```
Comment (l2silver) [inline web-client/src/pages/organization/member/[route].tsx:15]: I really like how you set this up! Comes together very easily.

File: web-client/src/pages/volunteer-toronto/member/[route].tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import Application from '../applications'
+import {
+    Container
+} from '@material-ui/core'
+
+const Dashboard: NextPage = ()=>{
+    const welcomeMsg = "Welcome back, Volunteer Toronto"
+    const components = [
+        {slug: 'applications', label:'Driver Applications', component: <Application />},
+        {slug: 'drivers-search', label:'Driver Pre-Screen', component: <DriversSearch />}
```
Comment (l2silver) [inline web-client/src/pages/volunteer-toronto/member/[route].tsx:17]: Can we give volunteer-toronto members the organization tab too? Thanks

File: web-client/src/shared/components/layout/LoadingRouter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, {useEffect} from 'react'
```
Comment (l2silver) [inline web-client/src/shared/components/layout/LoadingRouter/index.tsx:1]: So what does this do exactly?

File: web-client/src/shared/components/SignInPage/index.tsx [hunk 4]
```diff
@@ -19,23 +19,43 @@ interface FormData {
   password: string;
 }
 
+interface User {
+  domain:string
```
Comment (l2silver) [inline web-client/src/shared/components/SignInPage/index.tsx:23]: can we change domain to organization?

File: web-client/src/shared/components/SignInPage/index.tsx [hunk 5]
```diff
@@ -19,23 +19,43 @@ interface FormData {
...
+  user: User
+}
+
 const SignInPage: NextPage = () => {
 
   const { handleSubmit, register } = useForm<FormData>();
   const router = useRouter();
 
+  const createUrlFromData = ({domain, role}: User)=>{
+    return `/${domain ==='vt' ? 'volunteer-toronto': domain === 'org' ? 'organization' : 'individual'}/${role}`
```
Comment (l2silver) [inline web-client/src/shared/components/SignInPage/index.tsx:38]: Ah sorry, there is no individual space, that's just supposed to be the organization staff role

So there's 4 dashboards.
vt/admin
vt/member
organization/admin
organization/member

**Review by l2silver — CHANGES_REQUESTED** [review]
Amazing work! I can't believe you put this altogether.

File: web-client/src/pages/volunteer-toronto/member/[route].tsx (hunk 2 above)
Comment (Haris188) [reply web-client/src/pages/volunteer-toronto/member/[route].tsx:17]: Yes Leigh. But there is no organization component built. So should I just make a tab and leave it empty?

File: web-client/src/shared/components/layout/LoadingRouter/index.tsx (hunk 3 above)
Comment (Haris188) [reply web-client/src/shared/components/layout/LoadingRouter/index.tsx:1]: This basically redirects calls from /vt/admin, /vt/member, /org/admin, /org/member to driver-search tab

File: web-client/src/shared/components/SignInPage/index.tsx (hunk 5 above)
Comment (Haris188) [reply web-client/src/shared/components/SignInPage/index.tsx:38]: So which page would a organization staff role login lands on?

Comment (Haris188) [reply web-client/src/shared/components/SignInPage/index.tsx:38]: @l2silver Thanks for the review and appreciation!

File: web-client/src/pages/volunteer-toronto/member/[route].tsx (hunk 2 above)
Comment (l2silver) [reply web-client/src/pages/volunteer-toronto/member/[route].tsx:17]: Yep!

File: web-client/src/shared/components/layout/LoadingRouter/index.tsx (hunk 3 above)
Comment (l2silver) [reply web-client/src/shared/components/layout/LoadingRouter/index.tsx:1]: excellent

File: web-client/src/shared/components/SignInPage/index.tsx (hunk 5 above)
Comment (l2silver) [reply web-client/src/shared/components/SignInPage/index.tsx:38]: Lets have all the pages start on driver-search

Comment (Haris188) [reply web-client/src/shared/components/SignInPage/index.tsx:38]: So, It would be localhost/driver-search for individual login?

Comment (l2silver) [reply web-client/src/shared/components/SignInPage/index.tsx:38]: the individual login is organization/member/driver-search

**Review by l2silver — APPROVED** [review]

**Pull request merged**